CMakeLists.txt

```diff
@@ -40,13 +40,16 @@ endif()
 
 if(EOSERV_USE_UNITY_BUILD)
 	set(eoserv_SOURCE_FILES ${eoserv_UNITY_SOURCE_FILES})
+	set(eoserv_MAIN_FILES tu/main.cpp)
 else()
 	set(eoserv_SOURCE_FILES ${eoserv_ALL_SOURCE_FILES})
+	set(eoserv_MAIN_FILES src/main.cpp)
```

Comment:
Any reason to have `set(eoserv_MAIN_FILES tu/main.cpp)` inside both branches of the conditional instead of putting it just outside?

Reply:
It's different main files. One of them is tu (unity build version)